src/Sarif.Multitool/MergeCommand.cs

```diff
@@ -74,7 +77,7 @@ private static IEnumerable<string> GetSarifFiles(IEnumerable<string> files)
 				    directory = Path.GetDirectoryName(path) ?? path;
 				    filename = Path.GetFileName(path) ?? "*";
 			    }
-			    foreach (var file in Directory.GetFiles(directory, filename, SearchOption.AllDirectories))
+			    foreach (var file in Directory.GetFiles(directory, filename, searchOption))
 			    {
 				    if (file.EndsWith(".sarif", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
>InvariantCultureIgnoreCase [](start = 53, length = 26)

Use OrdinalIgnoreCase here instead

Reply:
Done.

---
In reply to: [141649198](https://github.com/Microsoft/sarif-sdk/pull/627#discussion_r141649198) [](ancestors = 141649198)